rsconf/package_data/rsconf/rsconf.sh

```diff
@@ -27,32 +27,42 @@ rsconf_append() {
 }
 
 rsconf_append_authorized_key() {
-    local user=$1
-    local key=$2
-    local ssh_d="$( getent passwd "$user" | cut -d: -f6 )"/.ssh
+    declare user=$1
+    declare key=$2
+    declare ssh_d="$( getent passwd "$user" | cut -d: -f6 )"/.ssh
     if [[ ! -e "$ssh_d" ]]; then
         install -d -m 700 -o "$user" -g "$user" "$ssh_d"
     fi
-    local keys_f=$ssh_d/authorized_keys
+    declare keys_f=$ssh_d/authorized_keys
     if [[ ! -e $keys_f ]]; then
         install -m 600 -o "$user" -g "$user" /dev/null "$keys_f"
     fi
     rsconf_edit_no_change_res=0 rsconf_append "$keys_f" "$key"
 }
 
+rsconf_clone_repo() {
+    declare repo=$1
+    declare dest=$2
+    declare user=$3
```

Comment:
user => owner (and implicitly group). 

Perhaps we need an install umask for this case. 

Also set the umask in a subshell and run git clone  In this case we can hardwire 027 as the umask for now until we need something different.

Note the weirdness about [git clone and umask](https://stackoverflow.com/a/11335009).

Reply:
9ffe221

Looks like that weirdness was fixed in [1.7.11.3](https://github.com/git/git/commit/45d4fdc2dc74c657a9c2e95bf04aed539fdcb0a4). We have 1.8.3.1 on centos7.